sklearn/svm/_classes.py

```diff
@@ -1155,6 +1155,8 @@ class SVR(RegressorMixin, BaseLibSVM):
     other :ref:`kernel_approximation`.
 
     Read more in the :ref:`User Guide <svm_regression>`.
+    See a visualization example in
+    :ref:`sphx_glr_auto_examples_svm_plot_svm_regression.py`.
```

Comment:
I have two comments regarding this.  The case for the `NuSVR` class is similar.

Firstly,

```suggestion
    See :ref:`sphx_glr_auto_examples_svm_plot_svm_regression.py`
    for a regression example using SVR with different kernels.
```

Secondly, the example should be moved to the end of the "Examples" section. If you scroll down in the `SVR` class doc-string, you’ll find the `Examples` section there.

Hopefully, the linting works out.